tests/containers/helm.pm

```diff
@@ -14,17 +14,33 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use mmapi 'get_current_job_id';
 
+
+sub install_kubectl {
+    my $version = get_var('KUBECTL_VERSION', 'v1.22.2');
+    assert_script_run("curl -LO https://dl.k8s.io/release/$version/bin/linux/amd64/kubectl");
+    assert_script_run("install -c -m 744 ./kubectl /usr/bin/kubectl");
```

Comment:
Oh, made a typo in my last comment, as maybe everyone should be able to execute this. Thus instead of 644 it should have been 755. But maybe we can keep it like this too :)